community/examples/ml-gke.yaml

```diff
@@ -48,18 +48,18 @@ deployment_groups:
         cidr_block: $(vars.authorized_cidr)
     outputs: [instructions]
 
-  # Docs at https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scripts/kubernetes-operations
```

Comment:
Are there any other examples using the Kubernets-operations module. If not we should probably delete all of the module documentation in our Repo. This would be under community/modules/kubernetes-operations/ and also from modules/README.md

Reply:
Deleted references to `kubernetes-operations` module, there is only one reference remaining which is around an output variable which is required by the `kubernetes-operations`

Remaining lines which I plan to clean up in a separate CL unless we should do this together as well:
*  https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/2af3c420f05970edb11f755af6556fde500b77d9/community/modules/scheduler/gke-cluster/README.md?plain=1#L159
* https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/2af3c420f05970edb11f755af6556fde500b77d9/community/modules/scheduler/gke-cluster/outputs.tf#L23